examples/api-tests/src/file-search.spec.js

```diff
@@ -14,10 +14,12 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+const { ConsoleOptions } = require('@theia/console/lib/browser/console-widget');
+
 // @ts-check
 describe('file-search', function () {
 
-    const { assert } = chai;
+    const { expect, assert } = chai;
```

Comment:
No real need for both `expect` and `assert`, you can do your tests with `assert` fine:

```ts
assert.equal(1, 1);
```